DEPRECATIONS.md

```diff
@@ -0,0 +1,20 @@
+# Deprecations
+
+## v1.6.0
+
+### stew.find
+
+(insert technical rationale that I am not fully aware of)
+
+Instead, you should use `broccoli-funnel`.
```

Comment:
I thought our discussion was that we merely deprecate `stew.find(x)` and allow `stew.find(x, { /* funnelOptions */)`

Reply:
Confirm, the PR was previous to the conversation :) I'll update tomorrow!

Reply:
ah ok 👍